modules/manage/partials/tiered-storage.adoc

```diff
@@ -1250,9 +1250,257 @@ endif::[]
 
 == Disaster recovery
 
-When you create a topic, you can use remote recovery to download the topic data from object storage. You can also use remote recovery to restore a topic that was deleted from a cluster.
+With tiered storage enabled, you can restore your cluster, including its metadata, in the event of failure. This is a simpler and less expensive alternative to active-active replication (for example, using xref:upgrade:migrate/data-migration.adoc[MirrorMaker 2]).
 
-CAUTION: Remote recovery is not intended to be a fully-functional disaster recovery solution. It is only safe when no other clusters are still writing to the specified bucket or container. Recovery does not provide snapshot-style consistency: some partitions in some topics will be more up to date than others. For example, restoring a consumer offsets topic could have offsets that point ahead of the restored data.
+You can take the following steps to restore a failed cluster:
+
+. Start a new cluster 
+. Restore data from a failed source cluster to the new cluster
+. Perform a rolling restart, if necessary
+
+Use this recovery method to restore your application to the latest functional state as quickly as possible. 
+
+CAUTION: Whole cluster restore is not intended to be a fully-functional disaster recovery solution. It does not provide snapshot-style consistency: some partitions in some topics will be more up to date than others. For example, restoring a consumer offsets topic could have offsets that point ahead of the restored data, although it will reflect the latest available offset in the topic data that is being restored. Committed transactions are not guaranteed to be atomic.
+
+=== Start a target cluster
+
+For Redpanda Enterprise, follow the steps to https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/[deploy a new cluster] in a self-hosted environment.
+
+For Redpanda Cloud, follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster]. 
+Cloud account credentials (such as secrets or access keys) are not included in backed up metadata. You must manually configure these credentials for the new cluster.
+
+=== Whole cluster restore
+
+Data from a failed cluster can be restored to a target cluster using an rpk command. 
+
+[,bash]
+----
+rpk cluster storage restore start
+----
+
+The following metadata is included in the restore:
+
+- Topic definitions. If you have enabled tiered storage only for specific topics, topics without tiered storage enabled will not be restored 
+- Users and access control lists (ACLs)
+- Schemas
+- The xref:develop:consume-data/consumer-offsets.adoc[consumer offsets topic]
+- All cluster configurations, including your Redpanda license key
```

Comment:
This isn't quite true, the list of ignored properties is here

https://github.com/redpanda-data/redpanda/blob/494689cfd1703cedc8c84d9bfbbbc203a8986025/src/v/cluster/cluster_recovery_reconciler.cc#L33-L48

The general theme of them is that we don't restore any configs that change the way the cluster interacts with tiered storage, since the cluster is already meant to be configured with tiered storage